build.gradle

```
@@ -13,6 +13,7 @@ dependencies {
     implementation 'org.projectlombok:lombok:1.18.22'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    testImplementation 'org.mockito:mockito-core:3.12.4'
```

```suggestion
    testImplementation 'org.mockito:mockito-core:5.2.0'
```
є новіша версія. 
Але я все одно не бачу де він використовується (я наведу приклад як його можна використати на наступному занятті)

---

я просто испытывал его ...